spec/features/user_votes_on_review_spec.rb

```diff
@@ -0,0 +1,72 @@
+# require 'rails_helper'
```

Comment:
This is a test we wrote for a future feature (voting on reviews) - if we hadn't commented it out, codeship would yell at us, and we'd end up in a catch 22. Should we just trash the test entirely for now?